### PR TITLE
Backport43 ccg client delete #1198948 keichwa

### PR DESCRIPTION
**Pull request by keichwa**
# Description

https://bugzilla.suse.com/show_bug.cgi?id=1198948
https://github.com/SUSE/spacewalk/issues/19385
BACKport from master with bells and whistles (trad. client)

# Target branches

Which documentation version does this PR apply to?

- [ ] Master (Default)
- [x] Manager-4.3
- [ ] Manager-4.2

# Links

Fixes #<insert issue or PR link, if any>

### CODE REVIEW

File: modules/client-configuration/pages/delete-clients.adoc [hunk 1]
```diff
@@ -19,13 +28,200 @@ For more information about the system set manager, see xref:client-configuration
...
+	        in case of error
+----
+
+Sometimes a new registration of a deleted (unregistered) client might not be possible.
+To solve this issue, some Salt cache files should be deleted on the {productname} Server (Salt master) before trying to re-register again:
+
+----
+rm /var/cache/salt/master/thin/version
+rm /var/cache/salt/master/thin/thin.tgz
+----
```
Comment (mcalmer) [inline modules/client-configuration/pages/delete-clients.adoc:77]: same remark as on the Head PR: maybe this is too prominent as in the majority of the cases thin is not used anymore.

**Review by mcalmer — APPROVED** [review]
Only the same remark as on Head. I leave it up to you if you still want to keep the sentence in this way.

File: modules/client-configuration/pages/delete-clients.adoc (hunk 1 above)
Comment (keichwa) [reply modules/client-configuration/pages/delete-clients.adoc:77]: thanks for the reminder, I add 4.3 here: https://github.com/uyuni-project/uyuni-docs/issues/2076

File: .changelog [hunk 2]
```diff
@@ -4,6 +4,9 @@
 # - Fixed error in Bat section of Upgrade Guide (bsc#1234567)
 # For guidelines: https://en.opensuse.org/openSUSE:Creating_a_changes_file_(RPM)#Changelog_section_.28.25changelog.29
 
+- Enhanced deleting clients in Client Configuration Guide
+- Warned about installing containerized proxy on traditional container host in
+  Installation and Upgrade Guide (bsc#1198948)
```
Comment (keichwa) [inline .changelog:9]:
```suggestion
```

**Review by 0rnela — APPROVED** [review]

**Pull request merged**